packages/maskbook/src/plugins/UnlockProtocol/constants.ts

```diff
@@ -0,0 +1,17 @@
+export const pluginIcon = '🔓'
+export const pluginName = 'Unlock Protocol'
+export const pluginDescription = 'Post content behind a secure paywall'
+export const identifier = 'com.maskbook.unlockprotocol'
+export const pluginId = 'maskbook.unlockprotocol'
+export const pluginMetaKey = 'com.metamask.unlockprotocol:1'
+
+export const graphEndpointKeyVal = {
+    1: 'https://api.thegraph.com/subgraphs/name/unlock-protocol/unlock',
```

Comment:
```suggestion
    [ChainId.Mainnet]: 'https://api.thegraph.com/subgraphs/name/unlock-protocol/unlock',
```

Reply:
Also check packages/web3-constants.

Reply:
```ts
const graphEndpointKeyVal: Record<ChainId, string> = {
   // ... your data set ...
}
```

Reply:
@iamzubin please resolve this

Reply:
@iamzubin Better use enumerate value here.